package/python-kiwi-spec-template

```diff
@@ -95,57 +109,74 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 # python2-kiwi
 %package -n python2-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
+License:        GPL-3.0+
 Group:          Development/Languages/Python
-Provides:       python-kiwi = %{version}-%{release}
-Provides:       kiwi-image:tbz
+Provides:       kiwi
+Obsoletes:      kiwi <= 9.0
 Provides:       kiwi-image:docker
 Provides:       kiwi-image:iso
-Provides:       kiwi-image:vmx
-Provides:       kiwi-image:pxe
 Provides:       kiwi-image:oem
+Provides:       kiwi-image:pxe
+Provides:       kiwi-image:tbz
+Provides:       kiwi-image:vmx
+Provides:       python-kiwi = %{version}-%{release}
+%if 0%{?fedora_version} || 0%{?suse_version}
 Recommends:     jing
-Requires:       update-alternatives
+%else
+Requires:       jing
+%endif
+Requires:       python-PyYAML
 Requires:       python-docopt
-Requires:       python-setuptools
-Requires:       python-lxml
-Requires:       python-xattr
-Requires:       python-six
 Requires:       python-future
-Requires:       python-PyYAML
+Requires:       python-lxml
 Requires:       python-requests
-Requires(post): update-alternatives
-Requires(postun): update-alternatives
+Requires:       python-setuptools
+Requires:       python-six
+Requires:       python-xattr
 # tools used by kiwi
 %if 0%{?suse_version}
-Requires:       zypper
+Requires:       update-alternatives
+Requires(post): update-alternatives
+Requires(postun): update-alternatives
+%ifarch x86_64
+Requires:       grub2-x86_64-efi
+%endif
+Requires:       qemu-tools
+Requires:       multipath-tools
 Requires:       squashfs
+Requires:       gptfdisk
+Requires:       zypper
```

Comment:
Please take a look into the expanded file, the diff fools you. It is wrapped. But what you gonna tell me is that you can use zypper with Fedora 26 ?